### ISSUE TITLE
The value of weight decay

### ISSUE THREAD

**Issue by yehuitang** [EleutherAI/pythia#132]
Thanks for you interesting work! What is the value of weight decay when training the model.  I note that it is 0.1 in the github (https://github.com/EleutherAI/pythia/blob/afec006b936a38131fc6c6d2b0a48425dd97bc6e/models/1B/pythia-1b.yml) but 0.01 in the paper (Table 6 in Appendix E). https://arxiv.org/pdf/2304.01373.pdf

Thanks!

**Comment by haileyschoelkopf** [comment]
Hi, this is an error in the paper, apologies--we used 0.1 for weight decay, matching the config files in this repo.
